content/01.summary.md

```diff
@@ -1,13 +1,14 @@
 ## Summary
-<!-- The summary is a single paragraph no longer than 150 words. An effective summary includes the following elements: (1) a brief background of the question that avoids statements about how a process is not well understood; (2) a description of the results and approaches/model systems framed in the context of their conceptual interest; and (3) an indication of the broader significance of the work. We discourage novelty claims (e.g., use of the word “novel”) because they are overused, tend not to add meaning, and are difficult to verify. Please do not include references in the summary. CURRENT WORD COUNT 148 -->
-With pediatric brain and spinal cord tumors being the leading disease-related cause of death in children, urgently identifying novel, curative therapeutic strategies for these tumors remains a critical need.
-To accelerate this process, the Children's Brain Tumor Network and Pacific Pediatric Neuro-Oncology Consortium have created a systematic process for tumor biobanking, model generation, and sequencing with immediate access to harmonized data.
-Here, we create the OpenPBTA project, in which we openly and collaboratively genomically characterize and molecularly subtype, to our knowledge, the largest cohort of pediatric brain tumor specimens and histologies to date.
-We establish over 40 scalable analysis modules, recapitulate key oncogenic drivers across histologies, and make the code and data openly available.  
-The OpenPBTA has become a foundational analysis platform that is actively being applied to study other pediatric cancers and inform molecular tumor board decision-making, making it an invaluable resource to the pediatric oncology community. 
-s
+<!-- The summary is a single paragraph no longer than 150 words. An effective summary includes the following elements: (1) a brief background of the question that avoids statements about how a process is not well understood; (2) a description of the results and approaches/model systems framed in the context of their conceptual interest; and (3) an indication of the broader significance of the work. We discourage novelty claims (e.g., use of the word “novel”) because they are overused, tend not to add meaning, and are difficult to verify. Please do not include references in the summary. CURRENT WORD COUNT 150 -->
+Pediatric brain and spinal cord tumors are the leading disease-related cause of death in children, and novel, curative therapeutic strategies for these tumors are urgently needed.
```

Comment:
grammatically this would need a semicolon since "thus" can't be used to form a compound sentence. - "Pediatric brain and spinal cord tumors are the leading disease-related cause of death in children **;** thus, novel, curative therapeutic strategies for these tumors are urgently needed."

Reply:
That sounds better!